src/workspaces/features/ArchiveWorkspaceDialog/ArchiveWorkspaceDialog.tsx

```diff
@@ -54,20 +55,30 @@ const ArchiveWorkspaceDialog = (props: ArchiveWorkspaceDialogProps) => {
   };
 
   return (
-    <Dialog open={open} onClose={onClose}>
+    <Dialog open={open} onClose={onClose} onClick={(e) => e.stopPropagation()}>
```

Comment:
What's the purpose of the onClick ?

Reply:
It was necessary on previous revisions to keep the dialog open but now it seems its not needed anymore so I removed, thanks